table-monday.tex

```diff
@@ -5,136 +5,190 @@
 \label{monday}
 \pagestyle{monday-table}
 \setPageBackground
-\noindent\begin{landscape}
+\noindent%\begin{landscape}
   \begin{center}
-    \noindent\begin{tabular}{Z{0.75cm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}}
+
+    %% monday - morning %%
+
+    \makebox[8cm] {
+    \noindent\hspace*{1.3cm}\begin{tabular}[t]{Z{0.75cm}Z{3.6cm};{0.2mm/0.4mm}Z{3.6cm};{0.2mm/0.4mm}Z{1.0cm}}
       \cellcolor{commongray}
-      & \multicolumn{1}{c}{\cellcolor{GHS} GHS}
-      & \multicolumn{1}{c}{\cellcolor{HSO} HSO}
-      & \multicolumn{1}{c}{\cellcolor{KHS} KHS*}
-      & \multicolumn{1}{c}{\cellcolor{Mathematikon C} Mathematikon C*}
+      & \multicolumn{1}{c}{\cellcolor{GHS} Großer Hörsaal}
+      & \multicolumn{1}{c}{\cellcolor{HSO} Hörsaal Ost}
```

Comment:
The booklet, the website and anything else using the Pretalx data use the abbreviations of the rooms. We cannot use the long names in the booklet because they take too much room on the description pages of the sessions.

Reply:
This is just a proposal. The room name abbreviations "GHS", "HSx", "KHS" are only informal. At the location, only the full names are signed on the room entrances. We can still change it on the website, pretalx and app(s), but in the booklet we need to decide now how we want to have it. I think it is better to print the actual room names.